### PR TITLE
add zshrc that was previously missing

### PR DESCRIPTION
**Pull request by lindsaylandry**

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -132,10 +137,16 @@ install_nodejs() {
...
   # install yarn
   log "⚠️  Installing Yarn"
-  curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version "$YARN_VERSION"
+  curl -o- -L https://yarnpkg.com/install.sh | sh -s -- --version "$YARN_VERSION"
   log "✅ Yarn installed"
 }
 
+# for using binaries acquired by go get
+git_config() {
+  git config --global url."git@github.com:".insteadOf https://github.com/
```
Comment (bhb603) [inline bootstrap.sh]: do we need this? I dont think i have this configured, but my `go gets` seem to work..

File: bootstrap.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```
[the remark this answers is not in the document]
Comment (lindsaylandry) [reply bootstrap.sh]: some people will be running this in zsh so i think it should be shell agnostic

File: bootstrap.sh [hunk 3]
```diff
@@ -107,7 +107,9 @@ install_ruby() {
   rbenv versions
 
   # shellcheck disable=SC2016
-  append_to_dotfile bash_profile 'eval "$(rbenv init -)"'
+  append_to_dotfile bash_profile "eval \"$(rbenv init -)\""
```
[the remark this answers is not in the document]
Comment (lindsaylandry) [reply bootstrap.sh]: sometimes there are differences between zshrc and bash_profile, but you're right, they are mostly the same

File: bootstrap.sh (hunk 1 above)
Comment (lindsaylandry) [reply bootstrap.sh]: oh, my comment was wrong. This is to enable bundler to work without having to specify a git token.

File: README.md [hunk 4]
```diff
@@ -55,6 +62,17 @@ source ~/.bash_profile
 * creates a recommended ssh config at `~/.ssh/config`
 * initializes `helm` client and the helpful [helm-diff](https://github.com/databus23/helm-diff) plugin
 
+## Bash on Catalina
```
Comment (scottrigby) [inline README.md]: There’s also an env var you can export to disable the annoying bash warnings on Catalina https://support.apple.com/en-us/HT208050

**Pull request merged**